misc/python/materialize/feature_benchmark/benchmark_versioning.py

```diff
@@ -16,7 +16,7 @@
 SHA256_BY_FRAMEWORK_DIR: dict[str, str] = {}
 SHA256_BY_FRAMEWORK_DIR[
     "."
-] = "839c67b7f4b38ea3cb55d57780fcbf636caa58545112f24c1ff5f8d6d934effa"
+] = "4f8719b8ff5db7aec30eab61699909981d165f61cdf5c2d7064ec435390c13df"
```

Comment:
What is this for? Why do we need to manually keep track of a hash?

Reply:
I introduced this with https://github.com/MaterializeInc/materialize/pull/27309 and https://github.com/MaterializeInc/materialize/pull/27312. Happy to discuss this. Though, I suggest to do this outside of this (unrelated) PR.

Reply:
I added it to our QA sync agenda.